torch/distributed/tensor/_ops/_tensor_ops.py

```diff
@@ -93,44 +93,59 @@ def propagate_single_input_strategy(op_schema: OpSchema) -> StrategyType:
 
 
 @register_op_strategy(aten.copy_.default)
-def copy_strategy(op_schema: OpSchema) -> StrategyType:
-    # TODO: this strategy is incorrect for copy_ in the case that src tensor
-    # is smaller rank than self tensor.  It is possible to select a strategy from self tensor
-    # that is invalid for dst tensor.
-    # It is also problematic to assume that shard(0) on src maps to shard(0) on self, since we
-    # may broadcast a new dim to the left or right of 0 when copying.
+def copy_inplace_strategy(op_schema: OpSchema) -> StrategyType:
```

Comment:
Hmmm I think you probably want to remove the copy strategy, just register it as a pointwise strategy.

Reply:
hmm. i would be ok with that, but does pointwise strategy properly enforce the extra requirement of inplace_ ops, where broadcasting 'self' to match 'src' is NOT allowed?

Reply:
Yeah, for inplace ops it just follow the first argument

Reply:
ok, yea i think we can try pointwise. Wish I had known about it!